ui/src/components/People/DetailsCard.vue

```diff
@@ -155,8 +160,22 @@
 import DynamicForm from "@/components/Form/DynamicForm.vue";
 import Practitioner from "@/mixins/Practitioner.js";
 import StructureDefinition from "@/mixins/StructureDefinition.js";
+import MobileLayout from "@/mixins/MobileLayout.js";
 
 export default {
+  computed:{
+    setDetailCardTitleStyle()
```

Comment:
Here is another example. Remove set from the method name. Also, you are in the detail card component so you don't need it in the computed property either. titleStyle for example.

Reply:
I use this computed property to set dynamically the title based on the screensize the title by default appear too big